### PR TITLE
change inputComponent to InputComponent

### PR DESCRIPTION
**Pull request by flyingcircle**
Fixes: #2089

### CODE REVIEW

File: src/input/Input.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ class Input extends React.Component {
       leftIconContainerStyle,
       rightIcon,
       rightIconContainerStyle,
-      inputComponent: InputComponent = TextInput,
+      InputComponent: InputComponent = TextInput,
```
Comment (iRoachie) [inline src/input/Input.js]:
```suggestion
      InputComponent = TextInput,
```

Comment (flyingcircle) [reply src/input/Input.js]: Thanks for the catch. I pushed another commit to resolve this.

File: docs/input.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ import { Input } from 'react-native-elements';
 - [`errorMessage`](#errormessage)
 - [`errorStyle`](#errorstyle)
 - [`errorProps`](#errorprops)
-- [`inputComponent`](#inputcomponent)
+- [`InputComponent`](#inputcomponent)
```
Comment (iRoachie) [inline docs/input.md]: Can you move this to the bottom of the list?  See how Button does it.

Comment (flyingcircle) [reply docs/input.md]: moved to the bottom

File: src/input/Input.js (hunk 1 above)
Comment (iRoachie) [reply src/input/Input.js]: You still need to change this

Comment (flyingcircle) [reply src/input/Input.js]: I tried that change, but the tests fail. I'm not sure why though. I thought that should provide that variable as a default? But it doesn't seem to.

Comment (iRoachie) [reply src/input/Input.js]: OKay I'd say put it as `InputComponent`,

Then add a default props section at the bottom.

```js
Input.defaultProps = {
  InputComponent: TextInput
}
```

Comment (iRoachie) [reply src/input/Input.js]: The tests also use snapshot testing. So you need to update the snapshots. See https://react-native-elements.github.io/react-native-elements/docs/testing.html#snapshot-testing

Comment (flyingcircle) [reply src/input/Input.js]: Alright got that piece working.

**Comment by iRoachie** [comment]
Thank you for your work 🚀

**Pull request merged**